core/components/molecules/sidesheet/__stories__/index.story.jsx

```diff
@@ -61,9 +55,8 @@ export const all = () => {
         Open Sidesheet
       </Button>
       <Sidesheet {...options}>
-        <Text>Modal Body</Text>
-        <ModalDescription {...modalDescriptionOptions} />
-        <ModalDescription {...modalDescriptionOptionsWithoutTitle} />
+        <SidesheetDescription {...sidesheetDescriptionOptions} />
+        <SidesheetDescription {...sidesheetDescriptionOptionsWithoutLabel} />
```

Comment:
Please rename the `sidesheetDescriptionOptions` & `sidesheetDescriptionOptionsWithoutLabel`